pkg/ovs/openflow/ofctrl_builder.go

```diff
@@ -340,6 +337,74 @@ func (b *ofFlowBuilder) MatchSCTPDstPort(port uint16) FlowBuilder {
 	return b
 }
 
+// MatchCTSrcIP matches the source IPv4 address of the connection tracker original direction tuple. This match requires
+// a match to valid connection tracking state as a prerequisite, and a valid connection tracking state matches include
+// "+new", "+est", "+rel" and "+trk-inv".
+func (b *ofFlowBuilder) MatchCTSrcIP(ip net.IP) FlowBuilder {
+	b.Match.CtIpSa = &ip
+	b.matchers = append(b.matchers, fmt.Sprintf("ct_nw_src=%s", ip.String()))
+	return b
+}
+
+func (b *ofFlowBuilder) MatchCTSrcIPNet(ipNet net.IPNet) FlowBuilder {
```

Comment:
I suggest a small comment here for symmetry:
> MatchCTSrcIPNet is the same as MatchCTSrcIP but supports IP masking.

Reply:
Done.